src/server/state.ts

```diff
@@ -229,6 +250,13 @@ export class GameState extends Schema implements IGameState {
     }
   }
 
+  private planPlayerColors(): void {
+    if (PLAYER_COLORS.length < 4) throw new Error('Not enough player colors in constants');
```

Comment:
Is it necessary? We are the one defining the PLAYER_COLORS constant.

Reply:
That's true